core/src/main/java/org/kestra/core/models/executions/Execution.java

```diff
@@ -62,7 +65,7 @@ public Execution withTaskRun(TaskRun taskRun) {
         );
 
         if (!b) {
-            throw new IllegalStateException("Can't replace taskRun '" +  taskRun.getId() + "' on execution'" +  this.getId() + "'");
+            throw new IllegalStateException("Can't replace taskRun '" + taskRun.getId() + "' on execution'" + this.getId() + "'");
```

Comment:
why not using string format ? this looks cool https://dzone.com/articles/java-string-format-examples